### PR TITLE
Add htdocs option

### LINKED ISSUE #3: Alternative to wp-content, one directory level up (i.e. htdocs)
First, this is a great tool - thank you.

While the `wp-content` option is great if your repo is based in `wp-content`, it can also be convenient to have your repository setup from the root of the project (e.g. `htdocs`).  This is particularly helpful when deploying a project via Git.

To this end, it would be nice to have an option that works just like the `wp-content` option except up a level.

### PR DESCRIPTION
**Pull request by JPry**
Fixes #3.

### CODE REVIEW

**Comment by JPry** [comment]
@salcode Just calling your attention to this. There's still a bit of work to be done around handling the downloading of WordPress to make sure it doesn't behave badly, but you're welcome to check out this branch and test it out. If you don't have a chance to before I add more to it, no worries.

At this point I haven't had a chance to directly test this with a full repo of my own, so if it's completely broken... I'm sorry!

~Disregard the failing Circle CI checks, as I haven't actually configured what Circle CI runs yet.~

**Comment by JPry** [comment]
@salcode I tested this with [your example repo](https://github.com/salcode/example-project-w-gitignore). It seemed to work, but I also don't see any conflicts with what WordPress provides by default.

File: README.md [hunk 1]
```diff
@@ -38,8 +38,9 @@ sites:
                 - twentyeleven
             delete_default_plugins: true
             delete_default_themes: true
-            wp-content: https://github.com/jquery/jquery-wp-content.git
+            wp_content: https://github.com/jquery/jquery-wp-content.git
             wp: true
+            htdocs: https://github.com/salcode/example-project-w-gitignore.git
```
Comment (salcode) [inline README.md:43]: @JPry I'm super flattered to have my repo included here. Thank you.

Comment (JPry) [reply README.md:43]: Thanks for having a public example I could use!

**Pull request merged**